### PR TITLE
feat(faucet): new faucet design

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: apps/faucet/src/components/Faucet.tsx [hunk 1]
```diff
@@ -20,6 +21,22 @@ interface ClaimInfo {
...
 const { serviceConfig, reCaptchaConfig } = environment;
 
+function CardHeading({
+  children,
+  className,
+}: PropsWithChildren<{ className?: string }>) {
+  return (
+    <h3
+      className={clsx(
+        'font-serif text-[20px] font-[500] uppercase leading-[26px] text-white',
```
Comment (olegshilov) [inline apps/faucet/src/components/Faucet.tsx]:
```suggestion
        'font-serif text-[20px] font-[500] leading-[26px] text-white',
```

File: apps/faucet/src/components/AccountInfo.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function AccountInfo() {
       )}
 
       {accBalance !== undefined && (
-        <div className="flex h-[40px] flex-1 flex-row items-center justify-end space-x-4 font-bold">
+        <div className="flex h-[40px] flex-1 flex-row items-center justify-end gap-x-4 font-serif font-bold">
```
Comment (olegshilov) [inline apps/faucet/src/components/AccountInfo.tsx]:
```suggestion
        <div className="flex h-[40px] flex-1 flex-row items-center justify-end gap-x-4 font-serif font-[500]">
```

File: apps/faucet/tailwind.config.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const { createGlobPatternsForDependencies } = require('@nx/react/tailwind');
 const { join } = require('path');
+const typographyPlugin = require('@tailwindcss/typography');
```
Comment (olegshilov) [inline apps/faucet/tailwind.config.js]: no need to use this plugin in faucet app

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/faucet/src/components/Faucet.tsx [hunk 4]
```diff
@@ -179,149 +195,146 @@ export function Faucet(): ReactElement {
...
+            <CardHeading>Connect wallet</CardHeading>
+
+            {currentChain?.unsupported && (
+              <div className="flex items-center">
+                <span
+                  className="text-haqq-orange hover:text-haqq-light-orange cursor-pointer text-sm leading-snug transition-colors duration-300"
+                  onClick={handleNetworkSwitch}
+                >
+                  Switch to {chain.name}
+                </span>
```
Comment (olegshilov) [inline apps/faucet/src/components/Faucet.tsx]: use `OrangeLink` component

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/faucet/tailwind.config.js (hunk 3 above)
Comment (hadzhehsen) [reply apps/faucet/tailwind.config.js]: Resolved

File: apps/faucet/src/components/AccountInfo.tsx (hunk 2 above)
Comment (hadzhehsen) [reply apps/faucet/src/components/AccountInfo.tsx]: Resolved

File: apps/faucet/src/components/Faucet.tsx (hunk 1 above)
Comment (hadzhehsen) [reply apps/faucet/src/components/Faucet.tsx]: Resolved

File: apps/faucet/src/index.html [hunk 5]
```diff
@@ -8,10 +8,10 @@
...
     <meta name="description" content="HAQQ Faucet" />
 
-    <link rel="shortcut icon" href="assets/favicon.svg" />
+    <link rel="shortcut icon" href="assets/favicon.ico" />
     <link
       rel="icon"
-      href="assets/favicon.svg"
+      href="assets/favicon.ico"
       sizes="any"
       type="image/svg+xml"
```
Comment (olegshilov) [inline apps/faucet/src/index.html]: `.ico` favicons must have different type

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/faucet/src/index.html (hunk 5 above)
Comment (hadzhehsen) [reply apps/faucet/src/index.html]: Resolved

File: apps/faucet/src/index.html [hunk 6]
```diff
@@ -1,30 +1,24 @@
...
 
-    <link rel="shortcut icon" href="assets/favicon.svg" />
-    <link
-      rel="icon"
-      href="assets/favicon.svg"
-      sizes="any"
-      type="image/svg+xml"
-    />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="HAQQ Faucet" />
```
Comment (olegshilov) [inline apps/faucet/src/index.html]:
```suggestion
  <meta name="description" content="HAQQ | Faucet" />
```

File: apps/faucet/src/index.html [hunk 7]
```diff
@@ -1,30 +1,24 @@
...
+  <base href="/" />
 
-    <link rel="shortcut icon" href="assets/favicon.svg" />
-    <link
-      rel="icon"
-      href="assets/favicon.svg"
-      sizes="any"
-      type="image/svg+xml"
-    />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (olegshilov) [inline apps/faucet/src/index.html]:
```suggestion
    <meta
      name="viewport"
      content="initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
    />
```

File: apps/faucet/src/index.html [hunk 8]
```diff
@@ -1,30 +1,24 @@
...
+  <meta name="description" content="HAQQ Faucet" />
+
+  <link rel="icon" href="/favicon.ico" sizes="any">
+  <link rel="icon" href="/icon.svg" type="image/svg+xml">
```
Comment (olegshilov) [inline apps/faucet/src/index.html]:
```suggestion
  <link rel="icon" href="/favicon.svg" type="image/svg+xml">
```

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/faucet/src/index.html (hunk 6 above)
Comment (hadzhehsen) [reply apps/faucet/src/index.html]: Resolved

File: apps/faucet/src/index.html (hunk 7 above)
Comment (hadzhehsen) [reply apps/faucet/src/index.html]: Resolved

**Review by olegshilov — APPROVED** [review]

**Pull request merged**